include/c++/v1/__threading_support

```diff
@@ -1,19 +0,0 @@
-///////////////////////////////////////////////////////////////////////////////
-// Licensed Materials - Property of IBM
-// ZOSLIB
-// (C) Copyright IBM Corp. 2022. All Rights Reserved.
-// US Government Users Restricted Rights - Use, duplication
-// or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
-///////////////////////////////////////////////////////////////////////////////
-
-#ifndef ZOS_LIBCPP_THREADING_SUPPORT
-#define ZOS_LIBCPP_THREADING_SUPPORT
-
-//TODO(itodorov) - zos: workaround for nanosleep WoZ conflict, revisit
```

Comment:
Is this fixed in the new Open XL 2.1? 

We needed this for a couple of C++ ports in z/OS Open Tools like duckdb